knowledge-base/chart-responsive.md

```diff
@@ -22,12 +22,28 @@ res_type: kb
 
 ## Description
 
-When the user resizes the browser window or some layout change happens (for example, a navigation panel is expanded or collapsed), you may want to have the charts redraw with the new dimensions.
+When the user resizes the browser window or some layout change happens (for example, a navigation panel is expanded or collapsed), you may want to have the Chart redraws with the new dimensions.
 
 ## Solution
 
-Generally, the `Width` and `Height` parameters of the chart can take values in `%`, and the chart will render according to the dimensions of its parent element.
+Generally, the `Width` and `Height` parameters of the Chart can take values in `%`, and the Chart will render according to the dimensions of its parent element.
 
-This works well for the initial rendering and the chart will be "responsive" immediately according to your layout, regardless of the display (desktop, tablet, phone).
+This works well for the initial rendering and the Chart will be "responsive" immediately according to your layout, regardless of the display (desktop, tablet, phone).
 
-When the layout changes dynamically at runtime, you have to call its `.Refresh()` method. You can find an example in the following sample project: [https://github.com/telerik/blazor-ui/tree/master/chart/responsive-chart](https://github.com/telerik/blazor-ui/tree/master/chart/responsive-chart)
+When the layout changes dynamically at runtime, you have to call its `.Refresh()` method. 
+
+## Sample project
+
+You can find an example in the following sample project: [https://github.com/telerik/blazor-ui/tree/master/chart/responsive-chart](https://github.com/telerik/blazor-ui/tree/master/chart/responsive-chart)
+
+The project is a Blazor Web App developed on the .NET 8 framework. It is created by following the [steps in the Telerik Blazor Web App (Tutorial)]({%slug getting-started/web-app%}). It features server interactive render mode and a global interactivity location. The sample project offers an exemplary project structure. While individual projects may vary, it's crucial to ensure that correct naming conventions and references are used. 
```

Comment:
Most of this paragraph is unnecessary and the end is hard to understand. If this is important, add it to the guidelines below.